### PR TITLE
IS-401: Implement IISS DB data related class

### PR DESCRIPTION
**Pull request by soobokjin**
Check Specific Roll & Responsibility of each class in jira

### CODE REVIEW

File: iconservice/base/msgpack_util.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 ICON Foundation
```
Comment (leeheonseung) [inline iconservice/base/msgpack_util.py]: use MsgPackForIpc instead of this

File: iconservice/iiss/database/iiss_batch.py [hunk 2]
```diff
@@ -17,18 +17,46 @@
...
+        else:
+            batch = IissBatch(self._db_iiss_tx_index)
+            self._iiss_batch_mapper[block_hash] = batch
+            return batch
+
+    # todo: rename method
+    def update_index_and_clear_mapper(self, block_hash: bytes) -> None:
+        if block_hash not in self._iiss_batch_mapper.keys():
+            # todo: set specific exception
+            raise Exception
```
Comment (leeheonseung) [inline iconservice/iiss/database/iiss_batch.py]: need to discussion

File: iconservice/iiss/database/iiss_db.py [hunk 3]
```diff
@@ -14,11 +14,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import TYPE_CHECKING
+
 import plyvel
 
-from ...database.db import KeyValueDatabase
+from iconservice.database.db import KeyValueDatabase
```
Comment (leeheonseung) [inline iconservice/iiss/database/iiss_db.py]:
```suggestion
from ...database.db import KeyValueDatabase
```

File: iconservice/iiss/iiss_data_converter.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# limitations under the License.
+
+from typing import Tuple, Any
+from ..base.msgpack_util import TypeTag, MsgPackConverter, Codec
+from ..base.address import AddressPrefix, Address
+from ..base.exception import InvalidParamsException
+from ..utils import int_to_bytes
+
+
+def address_to_bytes(addr: 'Address') -> bytes:
```
Comment (leeheonseung) [inline iconservice/iiss/iiss_data_converter.py]: use address.to_bytes_including_prefix, from_bytes_including_prefix

File: iconservice/iiss/iiss_data_converter.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        prefix_byte = int_to_bytes(addr.prefix.value)
+    return prefix_byte + addr_bytes
+
+
+def bytes_to_address(data: bytes) -> 'Address':
+    prefix = AddressPrefix(data[0])
+    return Address(prefix, data[1:])
+
+
+class IissCodec(Codec):
```
Comment (leeheonseung) [inline iconservice/iiss/iiss_data_converter.py]: we don't need to use it anymore

File: iconservice/iiss/iiss_data_converter.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        raise InvalidParamsException(f"Invalid encode type: {type(obj)}")
+
+    def decode(self, t: int, b: bytes) -> Any:
+        if t == TypeTag.ADDRESS:
+            return bytes_to_address(b)
+        else:
+            raise InvalidParamsException(f"UnknownType: {type(t)}")
+
+
+class IissDataConverter(MsgPackConverter):
```
Comment (leeheonseung) [inline iconservice/iiss/iiss_data_converter.py]: we don't need to use it anymore

File: iconservice/base/msgpack_util.py (hunk 1 above)
Comment (soobokjin) [reply iconservice/base/msgpack_util.py]: I have substituted MsgPackConverter with MsgPackForIpc

File: iconservice/iiss/database/iiss_db.py (hunk 3 above)
Comment (soobokjin) [reply iconservice/iiss/database/iiss_db.py]: I have fixed this code

File: iconservice/iiss/iiss_data_converter.py (hunk 5 above)
Comment (soobokjin) [reply iconservice/iiss/iiss_data_converter.py]: I have substituted MsgPackConverter with MsgPackForIpc

File: iconservice/iiss/iiss_data_converter.py (hunk 6 above)
Comment (soobokjin) [reply iconservice/iiss/iiss_data_converter.py]: I have substituted MsgPackConverter with MsgPackForIpc

**Pull request merged**